src/Avalonia.Controls/Window.cs

```diff
@@ -183,6 +183,15 @@ public WindowIcon Icon
             set { SetValue(IconProperty, value); }
         }
 
+        /// <summary>
+        /// Gets or sets the startup location of the window.
+        /// </summary>
+        public WindowStartupLocation WindowStartupLocation
```

Comment:
I [changed](https://github.com/AvaloniaUI/Avalonia/pull/1044/commits/e54d0c75d1c64f26be1aa6e67fede778729e0040#diff-fa616233ba8817223127ae63631cea21R198) it, but maybe I didn't understand the comment.